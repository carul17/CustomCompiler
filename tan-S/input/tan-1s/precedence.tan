main {
	var a := -5.5;
	a := a + (a + 7.5) - 6.0 - (7.0 / 7.0); # -10.5
	var b := 5 > 3 && 6 < 5;
	print a \n;
	print b \n;
}