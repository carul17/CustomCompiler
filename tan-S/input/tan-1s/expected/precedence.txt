-10.500000
true
